src/test/scala/MockedStreamsSpec.scala

```diff
@@ -77,14 +77,17 @@ class MockedStreamsSpec extends FlatSpec with Matchers {
   it should "assert correctly when processing multi input topology" in {
     import Fixtures.Multi._
 
-    val output = MockedStreams()
+    val builder = MockedStreams()
       .topology(topology1Output _)
       .input(InputATopic, strings, ints, inputA)
       .input(InputBTopic, strings, ints, inputB)
       .stores(Seq(StoreName))
-      .output(OutputATopic, strings, ints, expectedA.size)
+
+    val output = builder.output(OutputATopic, strings, ints, expectedA.size)
```

Comment:
no need for variables here :)

builder.output(OutputATopic, strings, ints, expectedA.size) shouldEqual expectedA

builder.stateTable(StoreName) shouldEqual inputA.toMap